internal/app/machined/pkg/controllers/config/k8s_control_plane_test.go

```diff
@@ -87,25 +87,10 @@ func (suite *K8sControlPlaneSuite) assertK8sControlPlanes(manifests []string) er
 	return nil
 }
 
-func (suite *K8sControlPlaneSuite) TestReconcileDefaults() {
+func (suite *K8sControlPlaneSuite) createMachine(cfg *config.MachineConfig) config.K8sControlPlaneAPIServerSpec {
```

Comment:
Better method name?

Reply:
`setupInputs`, as this is dependencies today, but it's going to be renamed to `Inputs` in #3405

Reply:
Renamed to `setupMachine`